extensions/python/PythonBindings.cpp

```diff
@@ -43,15 +44,16 @@ struct PyModuleDef minifi_module = {
 
 PyMODINIT_FUNC
 PyInit_minifi_native(void) {
-  const std::array<std::pair<PyTypeObject*, std::string_view>, 8> types = {
+  const std::array<std::pair<PyTypeObject*, std::string_view>, 9> types = {
```

Comment:
It may be better to use `std::to_array` so the element count template parameter wouldn't be needed to be updated every time the array is changed.
```
const std::array types = std::to_array<std::pair<PyTypeObject*, std::string_view>>({
  std::make_pair(PyLogger::typeObject(), "Logger"),
  std::make_pair(PyProcessSessionObject::typeObject(), "ProcessSession"),
  std::make_pair(PyProcessContext::typeObject(), "ProcessContext"),
  std::make_pair(PyProcessor::typeObject(), "Processor"),
  std::make_pair(PyScriptFlowFile::typeObject(), "FlowFile"),
  std::make_pair(PyRelationship::typeObject(), "Relationship"),
  std::make_pair(PyInputStream::typeObject(), "InputStream"),
  std::make_pair(PyOutputStream::typeObject(), "OutputStream"),
  std::make_pair(PyStateManager::typeObject(), "StateManager")
});
```

Reply:
good idea, I've included this in https://github.com/apache/nifi-minifi-cpp/pull/1530/commits/3d43afd375b5bd6b8eb7dc0941dbfffef1fb54cd#diff-9ab8cf198832c627eb6357bece7d822ae40cdef59fbd10881ffecfe7ffd5e93cR47-R57